README.md

```diff
@@ -123,6 +123,19 @@ This will call the /looker_auth endpoint and return a signed SSO URL that can be
 src=https://looker.example.com/embed/dashboards/11?sdk=2&embed_host=https://yourhost.example.com
 ```
 
+### Advanced Auth Configuration
+The Auth endpoint can be configured further, allowing custom Request Headers, as well as CORS support by passing an options object to the `init` method 
+
+```javascript
+LookerEmbedSDK.init('looker.example.com', 
+  {
+    url: 'https://api.acme.com/looker/auth',
+    headers: { 'Foo Header': 'Foo' },
+    params: { foo: 'bar' },
```

Comment:
Believe these need to be arrays of objects w/ a specific shape `Array<{name: string, value: string}>` like so: 
```suggestion
    headers: [{'name': 'Foo Header', 'value': 'Foo'}],
    params: [{'name': 'foo', 'value': 'bar'}],
```